requirements/dev.txt

```diff
@@ -4,6 +4,7 @@
 #
 #    pip-compile --output-file requirements/dev.txt requirements/base.in requirements/dev.in requirements/quality.in
 #
+--index-url http://localhost:3141/root/pypi/+simple/
```

Comment:
@jcdyer I think you have to add --no-index to avoid this.

Reply:
Thanks.